content/pt-br/docs/reference/glossary/taint.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Taint
+id: taint
+date: 2019-01-11
+full_link: /pt-br/docs/concepts/scheduling-eviction/taint-and-toleration/
+short_description: >
+  Um objeto principal consistindo de três propriedades principais: chave, valor, e efeito. `Taints` previnem o agendamento de pods em nós ou grupos de nós.
```

Comment:
Hi @stormqueen1990 In this case "previnem a alocação de pods" is better than "previnem o agendamento de pods". What do you think about this? Same for lines 14 and 18.

Reply:
Agreed. "Alocação" is indeed a better translation in this context.

Reply:
Hi @rodrigodelmonte!

Are you OK with replacing the words "agendar" and "agendamento" (when referring to Pod scheduling) throughout this glossary entry with "alocar" and "alocação"?

Reply:
Hi @MrErlison / @stormqueen1990 

I also agree with the suggestion :+1: